OpenRA.Utility/Program.cs

```diff
@@ -62,7 +62,8 @@ static void Main(string[] args)
 			{
 				var command = (IUtilityCommand)Activator.CreateInstance(commandType);
 				var kvp = new KeyValuePair<Action<ModData, string[]>, Func<string[], bool>>(command.Run, command.ValidateArguments);
-				actions.Add(command.Name, kvp);
+				if (!actions.ContainsKey(command.Name))
+					actions.Add(command.Name, kvp);
```

Comment:
It is probably a better idea to give these commands their own names as well as separating them by dll.